blocks/card-list-block/features/card-list/default.jsx

```diff
@@ -179,7 +181,7 @@ const CardListItems = (props) => {
               </article>
               {contentItems.slice(1).map((element) => {
                 const {
-                  headlines: { basic: headlineText } = {},
+                  headlines: { basic: headlineText },
```

Comment:
removed some of these fallbacks just bc they weren't tested and wanted to get a positive coverage quick

Reply:
Removing the `= {}` will mean that the headlines key is always available with basic key

Reply:
this is to just test a positive change in the github action

Reply:
see note in https://github.com/WPMedia/arc-themes-blocks/pull/1210